server/schedule/opt/healthy.go

```diff
@@ -21,18 +21,14 @@ import (
 var balanceEmptyRegionThreshold = 50
 
 // IsRegionHealthy checks if a region is healthy for scheduling. It requires the
-// region does not have any down or pending peers. And when placement rules
-// feature is disabled, it requires the region does not have any learner peer.
+// region does not have any down or pending peers.
 func IsRegionHealthy(cluster Cluster, region *core.RegionInfo) bool {
 	return IsHealthyAllowPending(cluster, region) && len(region.GetPendingPeers()) == 0
 }
 
 // IsHealthyAllowPending checks if a region is healthy for scheduling.
 // Differs from IsRegionHealthy, it allows the region to have pending peers.
```

Comment:
Prefer to rewrite the comment of this function.

Reply:
What would you suggest to add?